bin/compile

```diff
@@ -29,7 +29,7 @@ fi
 cache_get_url=$(curl -s https://api.heroku.com/apps/$SHARED_BUILD_CACHE_APP_NAME/build-metadata \
   -H "Accept: application/vnd.heroku+json; version=3.build-metadata" \
   -H "Authorization: Bearer $HEROKU_API_KEY" \
-  | ruby -e "require 'json'; print JSON[STDIN.read]['cache_get_url']")
+  | jq '.cache_get_url' | tr -d '"')
```

Comment:
```suggestion
  | jq -r '.cache_get_url')
```

Instead of piping `tr` to trim quotes, raw outputting the `jq` result should accomplish the same thing.

See documention on the raw output `-r` flag for `jq` → https://stedolan.github.io/jq/manual/#Invokingjq